pkg/api/operators/apis/studyjob/v1alpha1/studyjob_types.go

```diff
@@ -114,19 +114,25 @@ const (
 	OptimizationTypeMaximize OptimizationType = "maximize"
 )
 
-type GoTemplate struct {
+type TemplateSpec struct {
```

Comment:
This extension of studyjob API would be for v1alpha2? https://github.com/kubeflow/katib/issues/370